src/cryptography/x509/oid.py

```diff
@@ -92,6 +92,7 @@ class ExtensionOID(object):
     PRECERT_SIGNED_CERTIFICATE_TIMESTAMPS = (
         ObjectIdentifier("1.3.6.1.4.1.11129.2.4.2")
     )
+    ISSUING_DISTRIBUTION_POINTS = ObjectIdentifier("2.5.29.28")
```

Comment:
There's an OID for `ISSUING_DISTRIBUTION_POINT` on line 84 - do we need this one on line 95 too?

Reply:
Sure don't